src/test/java/io/xpring/xrpl/FakeXRPProtobufs.java

```diff
@@ -16,8 +16,8 @@
     static {
         try {
             testCurrencyCode = ByteString.copyFrom("123", "Utf8");
-        } catch (UnsupportedEncodingException e) {
-            e.printStackTrace();
+        } catch (UnsupportedEncodingException error) {
```

Comment:
Again, calling this `e` is a very normal thing in java, but if you must use a full word, let's call this `exception`